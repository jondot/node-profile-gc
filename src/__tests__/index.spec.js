import foobanzle from '../index'

describe('foobanzle', () => {
  it('with functions', async () => {
    const res = await foobanzle([1,2,3])
    expect({res}).toMatchSnapshot()
  })
})
