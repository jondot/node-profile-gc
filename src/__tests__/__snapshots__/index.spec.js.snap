// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`foobanzle with functions 1`] = `
Object {
  "res": "whoo : 1",
}
`;
